car/radio_interface.py

```diff
@@ -63,12 +77,36 @@ def run(self):
     def process_incoming(self, msg):
         if type(msg) == RaceStatus:
             logger.info("got race status message...{}".format(msg))
+            RaceFlagStatusEvent.emit(flag=RaceStatus.RaceFlagStatus.Name(msg.flagStatus))
+            if msg.flagStatus == RaceStatus.RED:
+                DriverMessageEvent.emit(text="Race Red Flagged", duration_seconds=10)
+            if msg.flagStatus == RaceStatus.BLACK:
+                DriverMessageEvent.emit(text="Race Black Flagged", duration_seconds=10)
+            if msg.flagStatus == RaceStatus.YELLOW:
+                DriverMessageEvent.emit(text="Course Yellow", duration_seconds=10)
```

Comment:
There's double duty going on here ... we emit a RaceFlagStatusEvent which will cause the background of the speedometer to change color.

But we also emit a specific message in the lower panel indicating the flag status of the race